Library/Homebrew/livecheck/strategy/page_match.rb

```diff
@@ -71,12 +71,29 @@ def self.page_matches(content, regex, &block)
 
         # Checks the content at the URL for new versions, using the provided
         # regex for matching.
-        sig { params(url: String, regex: T.nilable(Regexp)).returns(T::Hash[Symbol, T.untyped]) }
-        def self.find_versions(url, regex, &block)
+        #
+        # @param url [String] the URL of the content to check
+        # @param regex [Regexp] a regex used for matching versions in content
+        # @param provided_content [String] page content to use in place of
+        #   fetching via Strategy#page_content
+        # @return [Hash]
+        sig do
+          params(
+            url:              String,
+            regex:            T.nilable(Regexp),
+            provided_content: T.nilable(String),
+            block:            T.nilable(T.proc.returns(T.any(T::Array[String], String))),
```

Comment:
@reitermarkus I would appreciate if you could take a look at this to make sure the signature is correct. I'm still picking up Sorbet as I go along.